src/NuGet.Clients/NuGet.CommandLine/Commands/SourcesCommand.cs

```diff
@@ -2,18 +2,15 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Globalization;
 using System.Linq;
-using NuGet.Common;
-using NuGet.Configuration;
+using System.Runtime.CompilerServices;
+using NuGet.Commands;
+
+[assembly: TypeForwardedTo(typeof(NuGet.CommandLine.SourcesListFormat))]
```

Comment:
I don't know if this does much as nuget.exe doesn't ship as a library.

Reply:
yes, i could move all of these enums to nuget.commands.dll and nuget.commands namespace.
let me noodle. This is most compat, but is this compat work necessary?

Reply:
fixing.